<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddForm</class>
 <widget class="QWidget" name="AddForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>263</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Добавление элемента</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="btnHome">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../Загрузки/png-transparent-house-svg-vector-gray-house-gray-vector.png</normaloff>../../Загрузки/png-transparent-house-svg-vector-gray-house-gray-vector.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="btnInfo">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../Загрузки/png-clipart-computer-icons-information-button-button-interface-area-transformed.png</normaloff>../../Загрузки/png-clipart-computer-icons-information-button-button-interface-area-transformed.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labMarka">
         <property name="text">
          <string>Марка </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditMarka"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labYear">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Год выпуска</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditYear"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labNum">
         <property name="text">
          <string>Госномер</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditNum"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labSurn">
         <property name="text">
          <string>Фамилия владельца</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditSurn"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Загрузки/1828819.png</normaloff>../../Загрузки/1828819.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnHome</sender>
   <signal>clicked()</signal>
   <receiver>AddForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
